sensor_msgs/PointCloud2 input_pc
---
sensor_msgs/PointCloud2 response_pc

